jenkins/docker/cpp.Dockerfile

```diff
@@ -5,11 +5,18 @@ RUN apt-get install -y shellcheck
 RUN pip install gitlint
 
 # mongodb
+ARG DEBIAN_FRONTEND=noninteractive
+ENV TZ=Etc/UTC 
 RUN apt-get install -y wget gnupg \
-	&& wget -qO - https://www.mongodb.org/static/pgp/server-5.0.asc | apt-key add - \
+	&& wget -qO- https://www.mongodb.org/static/pgp/server-5.0.asc | gpg --dearmor > /etc/apt/trusted.gpg.d/mongo.gpg \
 	&& echo "deb [ arch=amd64,arm64 ] https://repo.mongodb.org/apt/ubuntu focal/mongodb-org/5.0 multiverse" \
 	| tee /etc/apt/sources.list.d/mongodb-org-5.0.list \
-	&& apt-get update \
+
+# mongodb requires ssl 1.1 but ubuntu 22.04 ships with ssl 3
+	&& wget http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2.16_amd64.deb \
```

Comment:
f => q ?

Reply:
Unfortunately, openssl dont have q on their download site.  This is only for MongoDB, Catapult will use 1.1.1q from the Conan packages.
http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/?C=M;O=D